client/dashboard/sites/settings-sftp-ssh/ssh-card.tsx

```diff
@@ -0,0 +1,304 @@
+import { CoreBadge } from '@automattic/components';
+import { DataForm } from '@automattic/dataviews';
+import { useQuery, useMutation } from '@tanstack/react-query';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	BaseControl,
+	Button,
+	Card,
+	CardBody,
+	ExternalLink,
+	SelectControl,
+	ToggleControl,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { sprintf, __ } from '@wordpress/i18n';
+import { trash } from '@wordpress/icons';
+import { store as noticesStore } from '@wordpress/notices';
+import { useMemo, useState } from 'react';
+import { useAuth } from '../../app/auth';
+import {
+	siteSshAccessEnableMutation,
+	siteSshAccessDisableMutation,
+	siteSshKeysQuery,
+	siteSshKeysAttachMutation,
+	siteSshKeysDetachMutation,
+	profileSshKeysQuery,
+} from '../../app/queries';
+import ClipboardInputControl from './clipboard-input-control';
+import type { SftpUser, SiteSshKey, ProfileSshKey } from '../../data/types';
+import type { Field } from '@automattic/dataviews';
+
+type SshCardFormData = {
+	connection_command: string;
+	ssh_key: string;
+};
+
+const SshKeyCard = ( {
+	siteSshKey,
+	userLocale,
+	isBusy,
+	onDetach,
+}: {
+	siteSshKey: SiteSshKey;
+	userLocale: string;
+	isBusy: boolean;
+	onDetach: ( siteSshKey: SiteSshKey ) => void;
+} ) => {
+	return (
+		<Card>
```

Comment:
The padding and border-radius are off on this card.

<img width="590" alt="Screenshot 2025-06-03 at 12 20 37" src="https://github.com/user-attachments/assets/c37321cc-cf08-47e8-899e-ec88966d10ae" />

We might be able to fix the padding issue by setting the `size` prop to `small`.

The border-radius is trickier as the component only appears to support a single `isRounded` prop. We typically reduce the border-radius of elements that are nested within a card. In this case we should use `radius-s` (2px) so the SSH key card matches the radius used on form inputs. If the only way to achieve this is a hacky CSS override, then perhaps we skip it. It would be good for the Card component to support this flexibility though.

Reply:
I think the CSS is the only way 🥲

Reply:
> We might be able to fix the padding issue by setting the size prop to small.

The padding for `size="small"` is currently set to `16px`, but the Figma design specifies `24px`. Is that acceptable?

Reply:
Yep, that's fine. 👍